### ISSUE TITLE
header titles not accessible

### ISSUE THREAD

**Issue by jmensch1** [react-navigation/react-navigation#7185]
I'm setting my screen titles using the `title` property in `navigationOptions`, and the titles aren't accessible to VoiceOver. 

I noticed that the `accessible` prop is explicitly set to `false` on the [HeaderTitle component](https://github.com/react-navigation/react-navigation-stack/blob/master/src/views/Header/HeaderTitle.js). If I change it to `true`, the titles become accessible. 

I'd submit a PR, but the prop was specifically set to `false` in this [commit](https://github.com/react-navigation/react-navigation-stack/commit/1931b5b9688007fa1316aef0023b6546a931e387). So I'm not sure if maybe there's some configuration where it's necessary for the titles not to be accessible.
